src/modules/navigation/ItemNavigation.tsx

```diff
@@ -44,13 +56,20 @@ const DrawerNavigation = (): JSX.Element | null => {
     return <LoadingTree />;
   }
 
+  if (shuffle) {
+    const baseId = rootId || '';
+    const memberId = member?.id || '';
+    const combinedUuids = combineUuids(baseId, memberId);
+    descendants = shuffleAllButLastItemInArray(descendants, combinedUuids);
+  }
+
   if (rootItem) {
     return (
       <MainMenu id={MAIN_MENU_ID}>
         <TreeView
           id={TREE_VIEW_ID}
           rootItems={[rootItem]}
-          items={[rootItem, ...(descendants || [])].filter(
+          items={[rootItem, ...descendants].filter(
```

Comment:
I think shuffling the descendants does not affect how they are rendered as it is based on the childrenOrder.

Reply:
I moved the ` || []` to line 37.

Not sure what you mean otherwise. Shuffling does work.